### ISSUE TITLE
Read variables list when using renku init

### ISSUE THREAD

**Issue by lorenzo-cavazzi** [SwissDataScienceCenter/renku-python#1126]
**Is your feature request related to a problem? Please describe.**
Users need to know the variable list required by a template to initialize a new project

**Describe the solution you'd like**
Once SwissDataScienceCenter/renku-project-template#68 is merged, we can 
* [x] Print the variables list in `--print-manifest` (renamed as `--list-templates`)
* [x] Remove `--description` (currently not used) since there is no need to have it as a special variable
* [x] Rename `--template-variables` to `--variables`
* [ ] [bonus]: when some variables are required but not specified, interactively ask them to the user -- we may move this to a new issue

**Describe alternatives you've considered**
We could scan all the files in the template and try to find all the jinja variables, but that requires checking out all the files and it doesn't provide a description.

**Additional context**
Listing the variables is needed also for implementing #862
